### DISCUSSION TITLE
A way to tell that someone has a NFT profile picture

### DISCUSSION THREAD

**Discussion by bbaster** [Docile-Alligator/Infinity-For-Reddit#1046] [Ideas]
A "NFT" badge could be shown on the profile page of users that have it set, for example next to the username or under the username next to the "Follow" button

**Comment by Wladefant** [comment]
There is not even an api request for that

**Reply by bbaster** [reply]
It might not be doable directly with the API, but it's certainly possible - direct URLs to normal Snoovatars look like this: `https://i.redd.it/snoovatar/avatars/....png` while direct URLs to NFT Snoovatars look like this: `https://i.redd.it/snoovatar/avatars/nftv2_bmZ0X2VpcDE1NToxMzdf....png`

**Reply by Wladefant** [reply]
Oh, ok
